<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window2</class>
 <widget class="QMainWindow" name="Window2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校园导航系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="VoicepushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>语音播报</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label
{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>                             南京邮电大学地图</string>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="BackpushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>530</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>返回上一级</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>71</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_2
{
	background-color: rgb(255, 255, 255);
	image: url(:/images/Map.jpg);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="StartFind">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>530</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopVoicepushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止播报</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>BackpushButton</zorder>
   <zorder>VoicepushButton</zorder>
   <zorder>StartFind</zorder>
   <zorder>StopVoicepushButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
